### PR TITLE
2fa and bugfixes for [game_chat]

### PR DESCRIPTION
**Pull request by minthe**

### CODE REVIEW

File: ms_game_chat/backend/backend_app/consumer/chat/user.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def get_avatar(self, user_id):
         if not user_exists:
             return None
         user_instance = MyUser.objects.get(user_id=user_id)  # changed id to user_id
-        avatar_url = user_instance.avatar.url if user_instance.avatar else None
+        avatar_url = user_instance.avatar if user_instance.avatar else None
```
Comment (minthe) [inline ms_game_chat/backend/backend_app/consumer/chat/user.py:123]: @mariekart42  fixes this error:

```
> File "/code/backend_app/consumer/chat/user.py", line 123, in get_avatar
>     avatar_url = user_instance.avatar.url if user_instance.avatar else None
```

File: ms_game_chat/backend/backend_app/urls.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
     # CHAT
     # 'user/avatar'
-    path('game/avatar/<str:username>/', views.uploadAvatar),
+    # path('game/avatar/<str:username>/', views.uploadAvatar),
```
Comment (minthe) [inline ms_game_chat/backend/backend_app/urls.py:28]: @mariekart42 function is not defined

File: ms_game_chat/backend/backend_app/views.py [hunk 3]
```diff
@@ -22,8 +22,8 @@ def createUser(request):
         jwt_user_id = request.user_id
 
         # check if user already exists
-        if not MyUser.objects.filter(user_id=jwt_user_id).exists():
-            return JsonResponse({'message': 'User does not exist'}, status=409)
+        if MyUser.objects.filter(user_id=jwt_user_id).exists():
+            return JsonResponse({'message': 'User already exists'}, status=409)
```
Comment (minthe) [inline ms_game_chat/backend/backend_app/views.py:26]: @mariekart42 fixed message and flipped the case

File: ms_game_chat/backend/requirements.txt [hunk 4]
```diff
@@ -3,4 +3,4 @@ psycopg[binary]
 channels~=4.0
 daphne~=4.0
 channels_redis~=4.1
-ft_jwt==0.1.0
\ No newline at end of file
+ft_jwt==0.1.2
```
Comment (minthe) [inline ms_game_chat/backend/requirements.txt:6]: @mariekart42  update to 1.0.2 improves how the time is calculated. before i was using deprecated methods

**Review by kris-bo — APPROVED** [review]

**Review by kris-bo — DISMISSED** [review]

**Review by mariekart42 — APPROVED** [review]

**Pull request merged**